chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertificationResultTestStandard.java

```diff
@@ -31,7 +31,7 @@
     /**
      * Test standard internal ID
      */
-    @XmlElement(required = true)
+    @XmlElement(required = false, nillable = true)
```

Comment:
Should this have been required = false all along?

Reply:
I don't even think it should be required = false ever, but based on the data that is being generated, we had domain objects where the value is empty.  Not sure how we should handle this situation.

Reply:
That's fine. Figured it was something that showed up as an error in your testing, but was curious